src/schema/ecommerce/initial_offer_mutation.ts

```diff
@@ -8,7 +8,7 @@ import { InitialOfferInputType } from "./types/initial_offer_input_type"
 import { moneyFieldToUnit } from "lib/moneyHelper"
 export const InitialOfferMutation = mutationWithClientMutationId({
   name: "InitialOffer",
-  description: "Creates an order with an artwork",
+  description: "Deprecated. Use AddInitialOfferToOrder instead.",
```

Comment:
I _think_ for this to be really deprecated (documentation-wise) we need to remove description and add `deprecationReason`, something like
```
deprecationReason: "Use AddInitialOfferToOrder instead.",
```
haven't used this for mutations but I think its the same.